daprdocs/content/en/developing-applications/building-blocks/bindings/howto-bindings.md

```diff
@@ -301,29 +273,25 @@ function sleep(ms) {
 }
 ```
 
-Navigate to the directory containing the above code, then run the following command to launch a Dapr sidecar and run the application:
-
-```bash
-dapr run --app-id orderprocessing --app-port 6001 --dapr-http-port 3601 --dapr-grpc-port 60001 npm start
-```
-
 {{% /codetab %}}
 
 {{< /tabs >}}
 
-All that's left now is to invoke the output bindings endpoint on a running Dapr instance.
+Invoke the output bindings endpoint on a running Dapr instance.
 
 You can also invoke the output bindings endpoint using HTTP:
 
 ```bash
 curl -X POST -H 'Content-Type: application/json' http://localhost:3601/v1.0/bindings/checkout -d '{ "data": 100, "operation": "create" }'
 ```
 
-As seen above, you invoked the `/binding` endpoint with the name of the binding to invoke, in our case its `checkout`.
-The payload goes inside the mandatory `data` field, and can be any JSON serializable value.
+As seen above:
+
+1. The example invoked the `/binding` endpoint with `checkout`, the name of the binding to invoke.
+1. The payload goes inside the mandatory `data` field, and can be any JSON serializable value.
+1. The `operation` field tells the binding what it needs to do.
```

Comment:
```suggestion
1. The `operation` field tells the binding what action it needs to take. For example the Kafka binding supports the `create` operation
```
For create above ^^ then add a link to https://docs.dapr.io/reference/components-reference/supported-bindings/kafka/#binding-support